### ISSUE TITLE
Installation command not working

### ISSUE THREAD

**Issue by 42elenz** [intel/llm-on-ray#110]
Hello! I tryed to run your repo, but the first command is not working 

> pip install .[cpu] -f https://developer.intel.com/ipex-whl-stable-cpu -f https://download.pytorch.org/whl/torch_stable.html

gives this error:

> ERROR: Ignored the following yanked versions: 1.11.100, 1.12.200
ERROR: Ignored the following versions that require a different python version: 3.10.0.0 Requires-Python >=2.7, !=3.0.*, !=3.1.*, !=3.2.*, !=3.3.*, <3.5; 3.7.4.2 Requires-Python >=2.7, !=3.0.*, !=3.1.*, !=3.2.*, !=3.3.*, <3.5
ERROR: Could not find a version that satisfies the requirement intel-extension-for-pytorch==2.1.0+cpu; extra == "cpu" (from llm-on-ray[cpu]) (from versions: 1.10.100, 1.11.0, 1.11.200, 1.12.0, 1.12.100, 1.12.300, 1.13.0, 1.13.100, 2.0.0, 2.0.100, 2.1.0, 2.1.100, 2.2.0)
ERROR: No matching distribution found for intel-extension-for-pytorch==2.1.0+cpu; extra == "cpu"

**Comment by carsonwang** [comment]
Did you start with a clean conda environment? If not, can you please create a new one?
You can also try the following command.
```
pip install .[cpu] --extra-index-url https://download.pytorch.org/whl/cpu --extra-index-url https://pytorch-extension.intel.com/release-whl/stable/cpu/us/
```

**Comment by xwu99** [comment]
Hi, this is due to upstream intel-extension-for-pytorch changed to a new repo. It will be fixed with #111. You can use the above @carsonwang method to install.

**Comment by xwu99** [comment]
We have fixed this, Pls check the updated README for using new pip index repo.
